packages/contracts/test/identityVAnchor/identity_vanchor.test.ts

```diff
@@ -1020,23 +1017,23 @@ describe('IdentityVAnchor for 2 max edges', () => {
     it('should reject tampering with extData relayer', async () => {
       const invalidExtData = aliceExtData;
       invalidExtData.relayer = toFixedHex('0x0000000000000000000000007a1f9131357404ef86d7c38dbffed2da70321337', 20),
-      await expect(
-        idAnchor.contract.transact({ ...publicInputs }, invalidExtData, { gasLimit: '0x5B8D80' })
-      ).to.be.revertedWith('Incorrect external data hash')
+        await expect(
```

Comment:
These spaces seem improperly added and also the format of these tests seems odd?

Why is there no semi-colon and instead a comma straight into a revert check? Maybe some aggregate formatting updates need to be fixed?